functions/lib/emails/email_reminder.py

```
@@ -12,127 +12,176 @@
+            send_email_with_loop(
+                logger,
+                purchaser.email,
+                combined_name.removeprefix(" and "),
+                event.eventName,
+                event.eventId,
+                event.startDate,
+                event.endDate,
+                event.location,
+            )
+
+    return https_fn.Response(f"Sent all reminders for upcoming events.")
```

Ruff (0.14.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)